<!DOCTYPE html><!--[if lt IE 8]><html lang="en" class="ie"><![endif]--><!--[if gt IE 8]><!--><html lang="en"><!--<![endif]--><head><meta charset="utf-8"><title>MOLGENIS - CoNVaDING</title><meta name="description" content="Copy Number Variation Detection in NGS Gene Panels"><meta name="author" content="MOLGENIS community"><meta name="handheldfriendly" content="true"><meta name="mobileoptimized" content="320"><meta name="viewport" content="width=device-width, initial-scale=1.0"><meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1"><meta http-equiv="cleartype" content="on"><script src="/res/js/jquery-1.11.3.min.js"> </script><script src="/res/js/bootstrap.min.js"></script><link rel="stylesheet" href="/res/css/prism.css"><link rel="stylesheet" href="/res/css/application.css" type="text/css"><link rel="shortcut icon" href="/favicon.png" type="image/png"><link rel="alternate" type="application/rss+xml" href="/blog/feed.xml" title="undefined"><script>(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)
[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

ga('create', 'UA-2273328-11', 'auto');
ga('send', 'pageview');</script></head><body><header class="molgenis-docs-header"><div role="navigation" class="navbar navbar-static-top"><div class="container container-fluid"><div class="navbar-header"><button type="button" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar" class="navbar-toggle collapsed"><span class="sr-only">Toggle navigation</span><span class="icon-bar"></span><span class="icon-bar"></span><span class="icon-bar"></span></button><a href="/" class="navbar-brand"><img src="/res/images/wordmark-blue.png" alt="undefined" class="wordmark"/></a></div><div id="navbar" class="collapse navbar-collapse">              <ul class="nav navbar-nav navbar-right"><li><a href="/start">Start</a></li><li><a href="/documentation">Documentation</a></li><li><a href="/compute">Compute</a></li><li class="active"><a href="/software">Software</a></li><li><a href="/develop">Develop</a></li><li><a href="/download">Download</a></li><li><a href="/about">About</a></li><li><a href="#searchbox" data-toggle="collapse"><i class="glyphicon glyphicon-search"></i></a></li></ul></div></div></div><div id="searchbox" class="collapse"><form role="search" action="/search-results"><input placeholder="Search" name="q" class="form-control"/></form></div><div class="molgenis-docs-titlebar active"><div class="container"> <h1>CoNVaDING</h1><p>Copy Number Variation Detection in NGS Gene Panels   </p></div></div></header><div class="molgenis-docs-body"><div class="container"><div class="row"><div role="complementary" class="col-md-3"><nav id="molgenis-docs-sidebar" class="molgenis-docs-sidebar hidden-print hidden-xs hidden-sm"><ul id="molgenis-docs-sidenav" class="nav molgenis-docs-sidenav"></ul></nav><script>$(function() {
    var sideNav = $("#molgenis-docs-sidenav");
    var index = 0;
    $(".row h1").each(function() {
        
        //create id for h1
        var id = (index++) + $(this).text().replace(/\s/gi,"-").replace(/\W/g,"").toLowerCase();
        $(this).prepend("<a name=\""+id+"\"></a>");
        
        //add h1 to list
        var li = $("<li><a href=\"#"+id+"\">"+$(this).text()+"</a></li>")
        sideNav.append(li);
        
        //wrap h1 into section
        var section = $(this).wrap("<section id=\""+id+"\"></section>").parent();
        
        //wrap parts of h1 into one section
        $(this).parent().nextUntil("h1").each(function(){            
            section.append($(this));
        });
        
        //repeat for h2
        if($("h2",$(this).parent()).length > 0) {
            var ul = $("<ul class=\"nav\"></ul>");
            li.append(ul);
            
            $("h2",$(this).parent()).each(function(){
            
                //create id for h2
                var id = (index++) + $(this).text().replace(/\s/gi,"-").replace(/\W/g,"").toLowerCase();
                $(this).prepend("<a name=\""+id+"\"></a>");
        
                //wrap h2 into section
                var section = $(this).wrap("<section id=\""+id+"\"></section>").parent();
        
                ul.append($("<li><a href=\"#"+id+"\">"+$(this).text()+"</a></li>"));
        
                //wrap parts of h2 into one section
                $(this).parent().nextUntil("h2").each(function(){            
                    section.append($(this));
                });
            

            });
        }
    });    
    
    //add "back to top"
    sideNav.append("<li><a href=\"#top\" style=\"font-size:smaller\">Back to top</a></li>");
    var $body   = $(document.body);
    
    //enable the 'active' highlighting using scrollspy plugin
    $('body').scrollspy({
        target: '#molgenis-docs-sidebar'
    });
    $('#molgenis-docs-sidebar').width($('#molgenis-docs-sidebar').parent().width());
    
    //enable affix
    $('#molgenis-docs-sidebar').affix({
        offset: {
            top: $('#molgenis-docs-sidebar').offset().top,
            bottom: $('.molgenis-docs-footer').outerHeight(true) + 10
        }
    });
    
    //resize affix box when window resizes
    $(window).resize(function () {
            $('.molgenis-docs-sidebar').width($('#molgenis-docs-sidebar').parent().width());
        });
    
    $("table").addClass("table table-bordered");
});</script></div><div role="main" class="col-md-9 contents"><h1>CoNVaDING User Guide</h1><p>Lennart Johansson <a href="&#x6d;&#97;&#x69;&#108;&#x74;&#111;&#58;&#x6c;&#46;&#106;&#x6f;&#104;&#x61;&#x6e;&#115;&#115;&#x6f;&#x6e;&#64;&#x75;&#x6d;&#x63;&#x67;&#x2e;&#110;&#108;">&#x6c;&#46;&#106;&#x6f;&#104;&#x61;&#x6e;&#115;&#115;&#x6f;&#x6e;&#64;&#x75;&#x6d;&#x63;&#x67;&#x2e;&#110;&#108;</a> &amp; Freerk van Dijk <a href="&#109;&#97;&#105;&#x6c;&#116;&#x6f;&#x3a;&#x66;&#46;&#118;&#97;&#x6e;&#46;&#100;&#x69;&#106;&#107;&#48;&#50;&#64;&#x75;&#x6d;&#99;&#x67;&#46;&#x6e;&#x6c;">&#x66;&#46;&#118;&#97;&#x6e;&#46;&#100;&#x69;&#106;&#107;&#48;&#50;&#64;&#x75;&#x6d;&#99;&#x67;&#46;&#x6e;&#x6c;</a>
v0.2, August 26th 2015:
Written for release CoNVaDING v0.1.4.17</p>
<h2>Introduction</h2><p>CoNVaDING (Copy Number Variation Detection In Next-generation sequencing Gene panels) was designed for small (single-exon) copy number variation (CNV) detection in high coverage next-generation sequencing (NGS) data, such as obtained by analysis of smaller targeted gene panels. </p>
<p>CoNVaDING makes use of a group of (at least 30) possible control samples from which the samples with the most similar overall pattern are selected as control samples. These samples are then used for read-depth normalization on all (autosomal) targets and on all targets per gene. CNV prediction is based on a combination of ratio scores and Z-scores of the sample of interest compared to the selected controlsamples.</p>
<p>Quality (QC) metrics are calculated per sample and per analyzed target. Output is generated on three levels:</p>
<ul>
<li><p>longlist: This list contains all calls, disregarding the target quality.</p>
</li>
<li><p>shortlist: This list contains a subset of the longlist, filtered on within sample target QC metrics.</p>
</li>
<li>final list: This list contains a subset of the shortlist, filtered on target QC metrics obtained from other samples.</li>
</ul>
<p>CoNVaDING has been written for use of CNV detection in high coverage NGS data (at least ~200x). With lower coverages it might still work, but more targets will fail QC metrics.</p>
<p>The program is written in perl and has dependencies on specific perl libraries as well as on samtools.</p>
<h2>Installation</h2><p>The latest version of CoNVaDING can be downloaded from [[X1]]DOWNLOADSITE[here].</p>
<p>CoNVaDING has several dependencies:</p>
<ul>
<li><p>[[X1]]<a href="https://www.perl.org/[Perl">https://www.perl.org/[Perl</a>]</p>
</li>
<li><p>The [[X1]]<a href="http://search.cpan.org/~mwendl/Statistics-Normality-0.01/lib/Statistics/Normality.pm[Statistics::Normality">http://search.cpan.org/~mwendl/Statistics-Normality-0.01/lib/Statistics/Normality.pm[Statistics::Normality</a> package] for perl  </p>
</li>
<li><p>[[X1]]<a href="http://www.htslib.org/[Samtools">http://www.htslib.org/[Samtools</a>]</p>
</li>
</ul>
<p>After installation the samtools executable has to be added to your local environment using the $PATH variable.</p>
<p>This version is known to be compatible with SAMtools version 0.1.18 and 0.1.19</p>
<h2>General comments</h2><p>Targets, or the region of interest, usually consist of an exon with some flanking bases, but can be anything that you specify, for instance part of an exon.</p>
<p>Because of the fact that the average coverage of the specified targets is used for the calculations, the resolution of the calls is also dependent on those targets. This means that the exact break point can not be determined. It might be within the last target of the call, or in the intron flanking this exon. Sub-exonic CNV&#39;s can also be missed if they if they do not have enough effect on the average coverage of the target.</p>
<h2>Analysis</h2><p>The analysis consists of three steps that have to be run separately and is based on a list of targets in a bed format.</p>
<p>CoNVaDING can be started using the following command:</p>
<p>  perl ./CoNVaDING-0.1.4.17.pl</p>
<p>If no options are used the help menu will appear.</p>
<p>  Usage: ./countCNV-v0.1.4.17.pl <mode> <parameters>
  -h            This manual.
  -mode            Mode to run in, one of the following required:
              StartWithBam :
                  Start with BAM files as input, to enable duplicate
                  removal use the rmdup variable.
                  REQUIRED:
                  [-inputDir, -outputDir, -bed, -controlsDir]
                  OPTIONAL:
                  [-rmDup, -useSampleAsControl]
              StartWithAvgCount :
                  Start with Average Count files as input. This is a five column text file
                  with predefined column names. Please read the manual for instructions.
                  REQUIRED:
                  [-inputDir, -outputDir, -bed, -controlsDir]
                  OPTIONAL:
                  [-useSampleAsControl]
              StartWithMatchScore :
                  Start with Normalized Coverage files as input.
                  REQUIRED:
                  [-inputDir, -outputDir, -controlsDir]
                  OPTIONAL:
                  [-controlSamples, -sexChr]
              StartWithBestScore :
                  Best score analysis using Match score files as input.
                  REQUIRED:
                  [-inputDir, -outputDir, -controlsDir]
                  OPTIONAL:
                  [-regionThreshold, -sexChr, -ratioCutOffLow, -ratioCutOffHigh, -zScoreCutOffLow, -zScoreCutOffHigh]
              GenerateTargetQcList :
                  Generate a target QC list to use as input for finallist creation.
                  REQUIRED:
                  [-inputDir, -outputDir, -controlsDir]
                  OPTIONAL:
                  [-controlSamples, -regionThreshold, -ratioCutOffLow, -ratioCutOffHigh, -zScoreCutOffLow, -zScoreCutOffHigh, -sampleRatioScore]
              CreateFinalList :
                  Creates the final list using the target QC list for filtering.
                  REQUIRED:
                  [-inputDir, -targetQcList, -outputDir]
                  OPTIONAL:
                  [-percentageLessReliableTargets]
  PARAMETERS:
  -inputDir        Input directory, depending on the analysis mode this contains
              BAM, AvgCount, normalized coverage or match score files.
  -bed            Input file specifying regions to analyze in BED format.
  -outputDir        Output directory to write results to.
  -controlsDir        Directory containing control samples.
  -targetQcList        Path to file containing target QC values.
  -controlSamples    Number of samples to use in Match score analysis. DEFAULT: 30
  -regionThreshold    Percentage of all control samples differing more than 3
              standard deviations from mean coverage of a region in the specified
              BED file to exlude from sample ratio calculation. DEFAULT: 20
  -rmDup        Switch to enable duplicate removal when using BAM files as input.
  -sexChr        Switch to include sex chromosomes in analysis.
  -useSampleAsControl    Switch to use samples as control. Example: when using BAM
              files to create count files and subsequentially use the
              generated count files as controls.
  -ratioCutOffLow    Lower ratio cutoff value. Region ratio values below this
              threshold are marked as deletion. DEFAULT: 0.65
  -ratioCutOffHigh    Higher ratio cutoff value. Region ratio values above this
              threshold are marked as duplication. DEFAULT: 1.4
  -zScoreCutOffLow    Lower Z-score cutoff value. Regions with a Z-score below
              this threshold are marked as deletion. DEFAULT: -3
  -zScoreCutOffHigh    Higher Z-score cutoff value. Regions with a Z-score above
              this threshold are marked as duplication. DEFAULT: 3
  -sampleRatioScore    Sample ratio z-score cutoff value. Sample with a ratio
              score below this value are excluded from analysis. DEFAULT: 0.09
  -percentageLessReliableTargets    Target labelled as less reliable in percentage
              of control samples. DEFAULT: 20</parameters></mode></p>
<p>Create normalized count files</p>
<pre><code>
The first step in the analysis is to create normalized count files. This can be done in two ways, from a bam file or from a text file including mean coverage per target.


StartWithBam
^^^^^^^^^^^^

If a bam file is used CoNVaDING will use samtools to calculate the mean coverage for each target. For this type of analysis the StartWithBam mode has to be selected.

The basic analysis starts as follows: 

  perl ./CoNVaDING-0.1.4.17.pl \
  -mode StartWithBam \
  -inputDir /PATH/TO/INPUTDIR \
  -controlsDir /PATH/TO/CONTROLSDIR \
  -outputDir /PATH/TO/OUTPUTDIR \
  -bed /PATH/TO/DIR/target_bedfile.bed

All bamfiles should be stored in the same folder, which can be specified with the &#39;inputDir&#39; option
The &#39;outputDir&#39; option should specify the path to the folder in which the normalized coverage files *.aligned.only.normalized.coverage.txt* should be stored.

The bed file should contain the regions of interest seperated in four columns specifying the chromosome, start position, stop position and the gene. No headers should be included.


It is important that the gene column has the exact same gene name for every target of the same gene, because these names are used to cluster targets for a normalization based on the targets belonging to the same gene. The bedfile should be sorted on chromosome and start position.

Bed file example: 

  2       96919506        96919893        TMEM127
  2       96920531        96920775        TMEM127
  2       96930836        96931159        TMEM127
  2       215593360       215593772       BARD1
  2       215595095       215595272       BARD1
  2       215609751       215609923       BARD1
  2       215610406       215610618       BARD1
  2       215617131       215617319       BARD1
  2       215632166       215632418       BARD1
  2       215633916       215634076       BARD1
  2       215645244       215646273       BARD1
  2       215656981       215657209       BARD1
  2       215661745       215661881       BARD1
  2       215674096       215674333       BARD1
  3       10183492        10183911        VHL
  3       10188158        10188360        VHL
  3       10191431        10191689        VHL


The analysis options can be further extended:

If a control set is not yet present, or if the samples that are analyzed have to be added to the control set the following options should be added:

  -useSampleAsControl
  -controlsDir /PATH/TO/CONTROLSDIR

The &#39;useSampleAsControl&#39; option specifies that the samples have to be used as a control sample later on. The &#39;controlsDir&#39; is the location where the normalized coverage files of the control samples will be stored.


If duplicates have to be removed before coverage calculations use the following option:

  -rmdup

This is advisable for capturing data, but should not be done for amplicon data.




StartWithAvgCount
^^^^^^^^^^^^^^^^^

If no bam files are present the analysis can also start with a text file specifying average counts per target. This file shoud contain the headers as shown in the example below.
This enables the use of alternative analysis software.


An example of a text file that can be used in this mode:

  CHR     START   STOP    GENE    REGION_COV
  2       96919506        96919893        TMEM127 209.606 
  2       96920531        96920775        TMEM127 230.959 
  2       96930836        96931159        TMEM127 127.735 
  2       215593360       215593772       BARD1   273.726 
  2       215595095       215595272       BARD1   297.522 
  2       215609751       215609923       BARD1   230.191
  2       215610406       215610618       BARD1   224.822
  2       215617131       215617319       BARD1   204.979 
  2       215632166       215632418       BARD1   211.352 
  2       215633916       215634076       BARD1   240.627 
  2       215645244       215646273       BARD1   281.97  
  2       215656981       215657209       BARD1   137.293 
  2       215661745       215661881       BARD1   264.81  
  2       215674096       215674333       BARD1   127.689 
  3       10183492        10183911        VHL     174.233 
  3       10188158        10188360        VHL     230.704 
  3       10191431        10191689        VHL     226.012 


When this mode is used the analysis is started as follows:

  perl ./CoNVaDING-0.1.4.17.pl \
  -mode StartWithAvgCount \
  -inputDir /PATH/TO/INPUTDIR \
  -outputDir /PATH/TO/OUTPUTDIR \
  -bed /PATH/TO/DIR/target_bedfile.bed


Also here the following options can be used if the samples should be used as a control set in later steps:

  -useSampleAsControl
  -controlsDir /PATH/TO/CONTROLSDIR

The -rmdup option is not available in this mode. If necessary the duplicates should have been removed before calculating the mean coverage per target.


Selecting the most informative control samples
~~~~~~~~~~~~~~~~~
</code></pre><p>StartWithMatchScore
^^^^^^^^^^^^^^^^^^^
The next step in the analysis is selecting the control samples estimated to be the most informative.</p>
<p>If a bam file is used CoNVaDING will use samtools to calculate the mean coverage for each target. For this type of analysis the StartWithBam mode has to be selected.</p>
<p>The basic analysis starts as follows: </p>
<p>  perl ./CoNVaDING-0.1.4.17.pl \
  -mode StartWithMatchScore \
  -inputDir /PATH/TO/INPUTDIR \
  -outputDir /PATH/TO/OUTPUTDIR \
  -controlsDir /PATH/TO/CONTROLSDIR</p>
<p>The &#39;inputDir&#39; option should specify the path to the folder in which the normalized coverage files <em>.aligned.only.normalized.coverage.txt</em> are be stored (the outputfolder of the previous step).</p>
<p>The &#39;outputDir&#39; option should specify the path to output folder. The script will produce two types of output files:</p>
<p><em>.best.match.score.txt</em> shows the matchscore and the paths to the selected control samples.</p>
<p><em>.normalized.autosomal.coverage.all.controls.txt</em> show the normalized coverage for all possible control samples</p>
<p>The &#39;controlsDir&#39; option should show the directory in which the control samples that have to be used are stored.</p>
<p>The analysis options can be further extended:</p>
<p>On default only targets located on autosomal chromosomes will be analyzed. If some targets are located on the sex chromosomes the following option should be added:</p>
<p>  -sexChr</p>
<p>Note that for this option only samples of the same sex as the sample of interest can be used as possible control samples.</p>
<p>On default 30 samples are selected to create the control group. If you wish to use a different number of control samples this can be indicated with the option:</p>
<p>  -controlSamples 40</p>
<p>to select for instance the 40 best matching samples.</p>
<p>CNV Detection
<del>~</del><del>~</del>~~~~</p>
<p>StartWithBestScore
^^^^^^^^^^^^^^^^^^
The last step in the analysis is the CNV detection itself.</p>
<p>The basic analysis starts as follows: </p>
<p>  perl ./CoNVaDING-0.1.4.17.pl \
  -mode StartWithBestScore \
  -inputDir /PATH/TO/INPUTDIR \
  -outputDir /PATH/TO/OUTPUTDIR \
  -controlsDir /PATH/TO/CONTROLSDIR</p>
<p>The &#39;inputDir&#39; option should specify the path to the folder in which the <em>.best.match.score.txt</em> files are stored (the outputfolder of the previous step).</p>
<p>The &#39;outputDir&#39; option should specify the path to output folder. The script will produce four types of output files:</p>
<p><em>.best.score.log</em> show the used control samples, the sample ratio score and the omitted regions for the sample ratio score calculation.</p>
<p><em>.best.score.longlist.txt</em> contains all calls, regardless of the target quality</p>
<p><em>.best.score.shortlist.txt</em> contains the high quality calls based on within sample target QC</p>
<p><em>.best.score.totallist.txt</em> contains information about all targets (ratio scores, Z-scores, QC) </p>
<p>The analysis options can be further extended:</p>
<p>The sample ratio calculation is based on calculation the variation coefficient of the normalized targets of the sample of interest. In this calculation highly variable targets are excluded. On default a target is considered highly variable if after transforming the normalized target ratio&#39;s of all samples in the possible control group 20 percent or more of the samples have a Z-score outside the -3 to 3 range. This percentage can be altered using the &#39;regionTreshold&#39; option.
For a threshold of 30 percent of the samples for instance the following option can be used:</p>
<p>  -regionThreshold 30</p>
<p>To alter the ratio thresholds when making a call for a deletion of duplication for a region during the analysis, the ratioCutOffLow and ratioCutOffHigh parameters can be used.
To apply a threshold of ratio score below 0.65 for a deletion and above 1.4 for duplication use:</p>
<p>  -ratioCutOffLow 0.65
  -ratioCutOffHigh 1.4</p>
<p>The same thresholds for calling a deletion or duplication can also be applied using the Z-score value cutoff.
To call a deletion when the Z-score is below -3 or duplication when the Z-score is above 3 use:</p>
<p>  -zScoreCutOffLow -3
  -zScoreCutOffHigh 3</p>
<p>To finetune the variant list one can generate a list of targets which in general are of lower quality in all possible controlsamples and apply this as a filter to generate a final list of high quality calls. This can be done by executing two steps:</p>
<p>GenerateTargetQcList
^^^^^^^^^^^^^^^^^^^^</p>
<p>To generate the list of targets and corresponding quality thresholds run:</p>
<p>  perl ./CoNVaDING-0.1.4.17.pl \
  -mode GenerateTargetQcList \
  -inputDir /PATH/TO/CONTROLSDIR \
  -outputDir /PATH/TO/OUTPUTDIR \
  -controlsDir /PATH/TO/CONTROLSDIR</p>
<p>For this analysis, the same region threshold, ratio cutoffs and Z-score cutoffs as explained above can be altered using their corresponding parameters.</p>
<p>CreateFinalList
^^^^^^^^^^^^^^^</p>
<p>To apply the generated list of sample target QCs to the <em>.best.score.shortlist.txt</em> files execute:</p>
<p>  perl ./CoNVaDING-0.1.4.17.pl \
  -mode CreateFinalList \
  -inputDir /PATH/TO/BESTSCOREOUTPUT \
  -targetQcList /PATH/TO/TARGETQCLISTFILE \
  -outputDir /PATH/TO/OUTPUTDIR</p>
<p>To change the percentage of samples in which a target can be labelled as less reliabe, for example in 20 percent of the samples, use the option:</p>
<p>  -percentageLessReliableTargets 20</p>
<p>This produces the following output file:</p>
<p><em>.finallist.txt</em> contains all final calls, basically a filtered shortlist file.</p>
<h2>Test dataset</h2><p>Not yet available</p>
<h2>QC Thresholds</h2><p>If the following thresholds are exceeded using default settings the CNV calling is less reliable.
The target ratio is also used to filter calls for the shortlist. Both QC metrics are used for filtering the final list.</p>
<p>Sample ratio: 0.09
Target ratio: 0.10</p>
<h2>Contact</h2><p>Mailto: </p>
<p>Lennart Johansson <a href="&#109;&#97;&#105;&#108;&#x74;&#111;&#x3a;&#x6c;&#x2e;&#106;&#111;&#104;&#x61;&#110;&#x73;&#x73;&#111;&#x6e;&#x40;&#x75;&#x6d;&#x63;&#x67;&#46;&#x6e;&#x6c;">&#x6c;&#x2e;&#106;&#111;&#104;&#x61;&#110;&#x73;&#x73;&#111;&#x6e;&#x40;&#x75;&#x6d;&#x63;&#x67;&#46;&#x6e;&#x6c;</a></p>
<p>Freerk van Dijk <a href="&#x6d;&#x61;&#x69;&#x6c;&#x74;&#x6f;&#58;&#x66;&#46;&#118;&#x61;&#110;&#46;&#x64;&#105;&#x6a;&#107;&#48;&#50;&#x40;&#x75;&#x6d;&#x63;&#103;&#x2e;&#110;&#x6c;">&#x66;&#46;&#118;&#x61;&#110;&#46;&#x64;&#105;&#x6a;&#107;&#48;&#50;&#x40;&#x75;&#x6d;&#x63;&#103;&#x2e;&#110;&#x6c;</a></p><center><br><i>See something that could be better? <a href="https://github.com/molgenis/molgenis.org/tree/master/public/software/CoNVaDING.md">Edit this page at Github</a></i><br></center></div></div></div></div><footer class="molgenis-docs-footer"><p>MOLGENIS and these docs are maintained by the <a href="about/team">MOLGENIS team and collaborators</a>. <br/> Documentation source code released under the <a href="/LICENSE-MIT">MIT License</a>, documentation contents under <a href="/LICENSE-CC">CC BY 3.0</a>, and MOLGENIS software code under the <a href="/LICENSE-LGPLv3">LGPLv3 License</a>.<ul class="text-muted"><li>Version 1.8.0</li><li>·</li><li><a href="https://github.com/molgenis/molgenis">MOLGENIS Github</a></li><li>·</li><li><a href="http://github.com/molgenis/molgenis/issues">MOLGENIS Issues</a></li><li>·</li><li><a href="https://github.com/molgenis/molgenis.org/tree/master/public">Docs Github    </a></li><li>·</li><li><a href="http://github.com/molgenis/molgenis.org/issues">Docs Issues</a></li><li>·</li><li><a href="https://github.com/molgenis/molgenis/releases">Releases</a></li></ul></p></footer><script src="/res/js/prism.js"></script></body></html>